mgmtsystem_action_efficacy/models/mgmtsystem_action.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2019 Marcelo Frare (Ass. PNLUG - Gruppo Odoo <http://odoo.pnlug.it>)
+# Copyright 2019 Stefano Consolaro (Ass. PNLUG - Gruppo Odoo <http://odoo.pnlug.it>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class MgmtsystemAction(models.Model):
+    """
+    Extend actions adding fields for record efficacy informations with changes tracking
+    """
+
+    _inherit = "mgmtsystem.action"
+
+    # new fileds
+    # value of efficacy
+    efficacy_value = fields.Integer(
+        "Rating",
+        help="0:not effective | 50:efficacy not complete | 100: effective",
+        track_visibility=True,
+    )
+    # user in charge of evaluation
+    efficacy_user_id = fields.Many2one(
+        "res.users",
+        "Inspector",
+        track_visibility=True,
+    )
+    # notes on the efficacy
+    efficacy_description = fields.Text("Notes")
+
+    @api.onchange("efficacy_value")
+    def _onchange_efficacy_value(self):
```

Comment:
Not specific to the migration, but rather to the original dev.
This should be in a `constraint`, not an `onchange`.

Reply:
Once approved, I will make a PR for it.